README.md

```diff
@@ -631,4 +634,11 @@ the mp4 or webm format. The id argument is the filename without the extension.
   :frame-height: 500
   :frame-width: 850
 ```
-
+###Available statuses
```

Comment:
The heading needs a space after the `#` characters. I'm not quite happy with the heading text. How about "List of exercise statuses".

Reply:
The list below should not be indented and there should be an empty line between the paragraph and the first list item. The explanations for the statuses could be expanded a little bit. E.g., "maintenance mode hides the exercise description and prevents submissions".